react-challenge.html

```diff
@@ -58,7 +61,9 @@ <h3>Mood Options:</h3>
       }
 
 
-
+// messaged sarah and went over the problems. The structure of the file and the component is fine as is. following the error messages which sarah said the first error I still couldnt understand the problem. 
```

Comment:
I appreciate you spent the time goin over this stuff. You got really close to figuring this problem out. Remember React only returns one thing at a time so we have to wrap everything up in <> </>.  Good stuff though Error logging is frustrating but highly useful.